Doc/library/dataclasses.rst

```diff
@@ -491,6 +491,22 @@ depend on one or more other fields.  For example::
         def __post_init__(self):
             self.c = self.a + self.b
 
+The :meth:`__init__` method generated by :func:`dataclass` does not call base class
+:meth:`__init__` methods. If the base class has a non-default :meth:`__init__` method, it is
```

Comment:
I don't know that anyone would know what "non-default `__init__` method" means. I'd just leave it as "an `__init__` method that needs to be called", or similar. Or maybe add words that say that in general, dataclass-generated `__init__` methods to not need to be explicitly called, because the derived class `__init__` takes care of initializing all fields.